be/src/vec/functions/array/function_array_aggregation.cpp

```diff
@@ -275,34 +312,57 @@ struct AggregateFunction<AggregateFunctionImpl<AggregateOperation::MAX>> {
 struct NameArraySum {
     static constexpr auto name = "array_sum";
 };
+struct NameArraySumDecimal256 {
+    static constexpr auto name = "array_sum_decimal256";
```

Comment:
```
#include "vec/aggregate_functions/aggregate_function.h"

use DECIMAL256_FUNCTION_SUFFIX instead of another literal.
```

Reply:
Hard to implement constexpr string concatenation.